tests/phpstan-tests.neon

```diff
@@ -1,6 +1,8 @@
 includes:
     - ../extension.neon
+    - ../vendor/nesbot/carbon/extension.neon
```

Comment:
What happens if you remove this line?

Reply:
The tests I added in Larastan pass with or without the setting.

Reply:
Regardless, we shouldn't be testing their extension in our tests---personally I'm of the mindset that Larastan should remove support for the Carbon mixins as Carbon provides their own extension for their own mixins

Reply:
I added a comment as to why I think it is useful for testing: https://github.com/larastan/larastan/pull/1857/commits/d60631110e12d7653c8d3986f568bc1eb927ca6b

Reply:
I still don't think this is necessary/required---our job is to test our own extension, not what other users might install in their application.

Reply:
I would argue that Larastan does not exist in isolation, thus a more integration-like test setup is useful.

Reply:
I could see that argument when running the e2e tests, but not when testing the code specific to this package